src/components/tree/CollapsibleTree.vue

```diff
@@ -78,29 +80,35 @@ export default Vue.extend({
     }
   },
   methods: {
-    startOpen () {
-      let toOpen = []
-      this.structure.map((item) => { if (item.open) toOpen[item.name] = true })
-      return toOpen
-    },
     selectElement (name) {
       this.$emit('input', name)
     },
     toggleCollapse (name) {
-      if (name in this.collapsed) {
-        delete this.collapsed[name]
+      if (this.collapsed === name) {
+        this.collapsed = ''
       } else {
-        this.collapsed[name] = true
+        this.collapsed = name
       }
       this.$forceUpdate()
     }
   },
-  components: { TransitionExpand, CollapseTreeIcon }
+  components: { TransitionExpand, CollapseTreeIcon, Spinner }
 })
 </script>
 
 <style scoped>
   [role="button"]{
     cursor: pointer;
   }
+  .parent-list{
+    font-weight: 600;
+  }
+  .child-list{
+    padding-left:35px;
+  }
+  .child-list.active{
+    background-color: rgb(108, 117, 125);
+    border-color: rgb(108, 117, 125);
```

Comment:
can we move this up to the app style for reuse in other components and have a more consistent ui

Reply:
used the css variables set by bootstrap